docs/deploy-rbd.md

```diff
@@ -27,7 +27,7 @@ make image-cephcsi
 **Available command line arguments:**
 
 | Option                | Default value         | Description                                                                                                                                                                                                                                                                          |
-|-----------------------|-----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --------------------- | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
not sure why it was not caught in CI. @humblec is it possible to update mdlinter to catch this one?

Reply:
I am not sure why it was caught on CI. may be a version difference of mdlinter.

Reply:
maybe, can you please check why it was not caught. if it's a mdlinter issue can you please update mdlinter to the same version where you are hitting this issue.